netlify.toml

```diff
@@ -620,4 +620,48 @@
 # Added: 2021-07-28
 [[redirects]]
     from = "/docs/self-host/deploy/qovery"
-    to = "/docs/self-host/deploy/other"
\ No newline at end of file
+    to = "/docs/self-host/deploy/other"
+# Added: 2021-08-02
+[[redirects]]
+    from = "/docs/api/index"
+    to = "/docs/api/overview"
+
+# Added: 2021-08-02
+[[redirects]]
+    from = "/docs/contribute/index"
```

Comment:
These `index` redirects shouldn't be in there. Suggests to me that `safe_redirects.py` isn't working as expected or something else has gone wrong.

Reply:
Actually, the `overview` files no longer exist. So I think this PR needs to be rebased of of `master`.

Reply:
You are correct! Rebase fixed it.

Reply:
😌